nnpdf_data/nnpdf_data/new_commondata/PHENIX_1JET_200GEV_ALL/filter.py

```diff
@@ -0,0 +1,89 @@
+import glob
+
+import pandas as pd
+import yaml
+
+
+def read_data(fnames):
+    df = pd.DataFrame()
+    for fname in fnames:
+        with open(fname, "r") as file:
+            data = yaml.safe_load(file)
+
+        pTbsub = data["independent_variables"][0]["values"]
+        pTsub = data["dependent_variables"][0]["values"]
+        ALLsub = data["dependent_variables"][1]["values"]
+
+        for i in range(len(ALLsub)):
+            df = pd.concat(
+                [
+                    df,
+                    pd.DataFrame(
+                        {
+                            "pT": [pTsub[i]["value"]],
+                            "pTmin": [pTbsub[i]["low"]],
+                            "pTmax": [pTbsub[i]["high"]],
+                            "eta": [0.0],
```

Comment:
```suggestion
                            "eta_min": [-0.35],
                            "eta_max": [0.35],
```
and propagate the information